frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -2692,4 +2694,18 @@ private void createModRequest(ResourceRequest request, ResourceResponse response
             response.setProperty(ResourceResponse.HTTP_STATUS_CODE, "500");
         }
     }
+
+    private Set<String> getProjectGroups(List<Project> projects) {
+        Set<String> businessUnit = new HashSet<String>();
```

Comment:
Better to have `TreeSet` instead of `HashSet` since it offers alphabetic sorting

Reply:
Used TreeSet for sorting.